generator/README.md

```diff
@@ -0,0 +1,1186 @@
+# Proxy Library RPC Generator
+
+## Overview
+
+This script provides a possibility to auto-generate Objective-C code (header \*.h and implementation \*.m classes) based on the SDL MOBILE_API XML specification provided as a source of true.
+
+## Requirements
+
+The script requires **Python 3.5** pre-installed on the host system. This is the minimal **Python 3** version that has not yet reached the end-of-life (https://devguide.python.org/devcycle/#end-of-life-branches).
```

Comment:
```suggestion
The script requires **Python 3** pre-installed on the host system. The minimal supported Python 3 version is **3.8.2**.  It may work on versions back to 3.5 (the minimal version that has not yet reached [the end-of-life](https://devguide.python.org/devcycle/#end-of-life-branches)), but this is not supported and may break in the future.
```